src/web/wms/src/main/resources/GeoServerApplication.properties

```diff
@@ -1,23 +1,31 @@
 #Generated by ResourceBundle Editor (http://eclipse-rbe.sourceforge.net)
 #pom.xml
 
-AbstractStylePage.confirmOverwrite = Really want to overwrite the current editor contents?
-AbstractStylePage.generate         = Generate ...
-AbstractStylePage.generateNew      = Generate a default style
-AbstractStylePage.copy             = Copy ...
-AbstractStylePage.copyFrom         = Copy from existing style
-AbstractStylePage.format           = Format
-AbstractStylePage.formatReadOnly   = Format only editable for new styles
-AbstractStylePage.name             = Name
-AbstractStylePage.styleFile        = Style file
-AbstractStylePage.upload           = Upload ...
-AbstractStylePage.validate         = Validate
-AbstractStylePage.preview          = Preview legend
-AbstractStylePage.styleNotFound    = Could not find Style file "{0}", please input a new style
-AbstractStylePage.genericError     = Invalid SLD
-AbstractStylePage.legend           = Legend
-ExtraStylesPalette.availableHeader = Available Styles
-ExtraStylesPalette.selectedHeader  = Selected Styles
+StyleAdminPanel.styleDataSpan       = Style Data
```

Comment:
Yes there are other language files, but I think there is enough new content that we cannot readily migrate them with sufficient accuracy.